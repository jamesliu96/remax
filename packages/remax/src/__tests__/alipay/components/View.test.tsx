import * as React from 'react';
import TestRenderer from 'react-test-renderer';
import View from '../../../adapters/alipay/components/View';

describe('View', () => {
  it('render correctly', () => {
    const callback = () => {};

    const testRenderer = TestRenderer.create(
      <View onAnimationStart={callback} catchClick={callback} className="class">
        view
      </View>
    );

    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
