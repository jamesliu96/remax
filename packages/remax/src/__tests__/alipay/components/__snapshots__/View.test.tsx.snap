// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View render correctly 1`] = `
<view
  catchTap={[Function]}
  class="class"
  onAnimationStart={[Function]}
>
  view
</view>
`;
